versioned_docs/version-v1.1/upload-image.md

```diff
@@ -64,4 +64,38 @@ You can add labels to the image, which will help identify the OS type more accur
 
 If your image name or URL contains any valid information, the UI will automatically recognize the OS type and image category for you. If not, you can also manually specify those corresponding labels on the UI.
```

Comment:
```suggestion
The Harvester UI can extract information such as OS type and file format from image names and URLs. You can also manually specify the same information on the UI.
```

Reply:
I rewrote this paragraph for the Harvester PoC Guide. I'd like to apply the change here as well.

Reply:
@jillian-maroket 
I think maybe it's better to create another PR for this. 
WDYT? @bk201, thanks.

Reply:
Yeah, I think it's better to create another PR and include all supported versions.

Reply:
@WebberHuang1118 @bk201 In that case, I'll save this for later. We can create a PR for non-release content enhancements.